client/app.js

```diff
@@ -5,6 +5,5 @@ jQuery = $;
 
```

Comment:
bleh, oops. All these files got added into the one commit, wasn't paying enough attention I guess.